setup.py

```diff
@@ -48,11 +48,27 @@
     extra_compile_args=['-Wall', '-Werror'],
 )
 
+if "RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE" in os.environ:
+    # borrow a similar hack from dask-cuda: https://github.com/rapidsai/dask-cuda/blob/b3ed9029a1ad02a61eb7fbd899a5a6826bb5cfac/setup.py#L12-L31
+    orig_get_versions = versioneer.get_versions
+
+    version_override = os.environ.get("RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE", "")
+
+    def get_versions():
+        data = orig_get_versions()
+        if version_override != "":
+            data["version"] = version_override
+        return data
```

Comment:
I think that we want to complain more loudly if something is in the environment but empty. So how about writing this like so:

```python
if "RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE" in os.environ:
    orig_get_versions = versioneer.get_version

   version_override = os.environ["RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE"]
   if version_override == "":
       raise RuntimeError("RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE in environment but empty")
   
   def get_versions():
       data = orig_get_versions():
       data["version"] = version_override
       return data

    versioneer.get_versions = get_versions
```

Reply:
Or is it that we only need this for nightlies and for actual tagged version releases we don't override things?

Reply:
It's a good suggestion, I will implement it. As I mentioned above, right now we always override (as a bandaid), so it's better to be safe about it.